.pre-commit-config.yaml

```diff
@@ -3,52 +3,42 @@ repos:
     rev: 20.8b1
     hooks:
       - id: black
-        args: ['--quiet']
+        args: ["--quiet"]
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v3.2.0
+    rev: v3.4.0
     hooks:
       - id: check-case-conflict
       - id: end-of-file-fixer
       - id: trailing-whitespace
       - id: check-ast
       - id: check-json
       - id: check-merge-conflict
-      - id: detect-private-key
-      - id: double-quote-string-fixer
   - repo: https://gitlab.com/pycqa/flake8
-    rev: 3.8.3
+    rev: 3.8.4
     hooks:
       - id: flake8
-        additional_dependencies: [
-            'flake8-bugbear==20.1.4',  # Looks for likely bugs and design problems
-            'flake8-comprehensions==3.2.3',  # Looks for unnecessary generator functions that can be converted to list comprehensions
-            'flake8-deprecated==1.3',  # Looks for method deprecations
-            'flake8-use-fstring==1.1',  # Enforces use of f-strings over .format and %s
-            'flake8-print==3.1.4',  # Checks for print statements
-            'flake8-docstrings==1.5.0',  # Verifies that all functions/methods have docstrings
-            'flake8-type-annotations==0.1.0',  # Looks for misconfigured type annotations
-        ]
   - repo: https://github.com/asottile/pyupgrade
-    rev: v2.7.2
+    rev: v2.7.4
     hooks:
       - id: pyupgrade
         args: ["--py36-plus"]
-  - repo: https://github.com/pycqa/isort
-    rev: 5.5.2
+  - repo: https://github.com/asottile/seed-isort-config
+    rev: v2.2.0
+    hooks:
+      - id: seed-isort-config
+  - repo: https://github.com/pre-commit/mirrors-isort
+    rev: "v5.7.0"
```

Comment:
Both of these repos say they're deprecated and that one should use `isort` directly. Unless you have a good reason to change this, I think it should keep the current config.

Reply:
done